pkg/common/restfuladapter/adapter.go

```diff
@@ -0,0 +1,16 @@
+package restfuladapter
+
+import (
+	"github.com/emicklei/go-restful"
```

Comment:
is this now the only package reliant on go-restful?

is there a golang way to hide this so that we don't end up with a transitive dependency for all those relying on this library?